packages/framework/fluid-framework/README.md

```diff
@@ -2,6 +2,13 @@
 
 The `fluid-framework` package bundles a collection of Fluid Framework client libraries for easy use when paired with a corresponding service client package (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluid-experimental/osdp-client (BETA)`).
 
+This package reexports these APIs from other Fluid client packages reducing the number of direct package dependencies needed.
+There are some packages there are not included as part of this `fluid-framework` package which users may want to use with this package:
+
+-   A service client (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluid-experimental/osdp-client (BETA)`) to allow connecting to a Fluid service.
```

Comment:
Moving tinylicious client to the end and adding a note so people don't think it's "as good/prod-ready" as the other two.
```suggestion
-   A service client (e.g. `@fluidframework/azure-client`, `@fluid-experimental/osdp-client (BETA)`, or `@fluidframework/tinylicious-client` for local development) to allow connecting to a Fluid service.
```